ddsrouter_yaml/include/ddsrouter_yaml/validator/ddsrouter_config_schema.json

```diff
@@ -0,0 +1,477 @@
+                                }
+                            }
+                        },
+                        "required":[
+                            "password",
+                            "cert"
+                        ]
+                    }
+                },
+                {
+                    "if":{
+                        "properties":{
+                            "cert":{
+                                "type":"string"
+                            }
+                        },
+                        "required":[
+                            "cert"
+                        ]
+                    },
+                    "then":{
+                        "properties":{
+                            "cert":{
+                                "not":{
+                                    "pattern":"^\\s*$"
+                                }
+                            }
+                        },
+                        "required":[
+                            "password",
+                            "private_key"
+                        ]
+                    }
+                }
+            ],
+            "title":"TLS"
+        },
+        "Address":{
+            "type":"object",
+            "additionalProperties":false,
+            "properties":{
+                "ip":{
+                    "type":"string",
+                    "anyOf":[
+                        {
+                            "format":"ipv4"
+                        },
+                        {
+                            "format":"ipv6"
+                        }
+                    ]
+                },
+                "domain":{
+                    "type":"string"
+                },
+                "port":{
+                    "type":"integer",
+                    "minimum":0,
+                    "maximum":65535
+                },
+                "transport":{
+                    "type":"string",
+                    "enum":[
+                        "tcp",
+                        "udp"
+                    ]
+                },
+                "ip-version":{
+                    "type":"string",
+                    "enum":[
+                        "ipv4",
+                        "ipv6"
+                    ]
+                }
+            },
+            "allOf":[
+                {
+                    "anyOf":[
+                        {
+                            "required":[
+                                "domain"
+                            ]
+                        },
+                        {
+                            "required":[
+                                "ip"
+                            ]
+                        }
+                    ]
+                },
+                {
+                    "required":[
+                        "port"
+                    ]
+                }
+            ],
+            "title":"Address"
+        }
+    }
```

Comment:
I would add `Domain` as its own object, so we can set a minimum and maximum values.

Reply:
If it is used only in simple, we can add maximum and minimum values directly in domain property

Reply:
I thought the intention of the validation procedure was that of verifying the structure and types are correct, without checking concrete values in this stage.